src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -306,25 +338,50 @@ def disable_project_uptime_subscription(uptime_monitor: ProjectUptimeSubscriptio
     # All project subscriptions are disabled, we can disable the subscription
     # and remove the remote subscription.
     if not has_active_subscription:
-        uptime_subscription.update(status=UptimeSubscription.Status.DISABLED.value)
+        uptime_subscription.status = UptimeSubscription.Status.DISABLED.value
+        uptime_subscription.save()
```

Comment:
Why are we changing this?

Reply:
Using `.update` does not update the instance of the model without a `refresh_from_db`. Which isn't great when we create the monitor and immediately disable it

Reply:
It does - you're thinking of `UptimeSubscription.objects.update(...)`. The model method is here: https://github.com/getsentry/sentry/blob/646521b0759c9b369ab65332cb2767cc573b653a/src/sentry/db/models/query.py#L86-L122

Reply:
er.. hmmm. my tests were not reflecting this 😬

Reply:
nvm, this does work. not sure what I was seeing.